projects/plugins/jetpack/extensions/blocks/ai-assistant/components/ai-assistant-controls/index.tsx

```diff
@@ -52,36 +53,61 @@ const QUICK_EDIT_KEY_MAKE_LONGER = 'make-longer' as const;
 // Ask AI Assistant option
 export const KEY_ASK_AI_ASSISTANT = 'ask-ai-assistant' as const;
 
-const quickActionsList = [
-	{
-		name: __( 'Correct spelling and grammar', 'jetpack' ),
-		key: QUICK_EDIT_KEY_CORRECT_SPELLING,
-		aiSuggestion: PROMPT_TYPE_CORRECT_SPELLING,
-		icon: termDescription,
-	},
-	{
-		name: __( 'Simplify', 'jetpack' ),
-		key: QUICK_EDIT_KEY_SIMPLIFY,
-		aiSuggestion: PROMPT_TYPE_SIMPLIFY,
-		icon: post,
-	},
-	{
-		name: __( 'Summarize', 'jetpack' ),
-		key: QUICK_EDIT_KEY_SUMMARIZE,
-		aiSuggestion: PROMPT_TYPE_SUMMARIZE,
-		icon: postExcerpt,
-	},
-	{
-		name: __( 'Expand', 'jetpack' ),
-		key: QUICK_EDIT_KEY_MAKE_LONGER,
-		aiSuggestion: PROMPT_TYPE_MAKE_LONGER,
-		icon: postContent,
-	},
-];
+const quickActionsList = {
+	default: [
+		{
+			name: __( 'Correct spelling and grammar', 'jetpack' ),
+			key: QUICK_EDIT_KEY_CORRECT_SPELLING,
+			aiSuggestion: PROMPT_TYPE_CORRECT_SPELLING,
+			icon: termDescription,
+		},
+	],
+	'core/paragraph': [
+		{
+			name: __( 'Simplify', 'jetpack' ),
+			key: QUICK_EDIT_KEY_SIMPLIFY,
+			aiSuggestion: PROMPT_TYPE_SIMPLIFY,
+			icon: post,
+		},
+		{
+			name: __( 'Summarize', 'jetpack' ),
+			key: QUICK_EDIT_KEY_SUMMARIZE,
+			aiSuggestion: PROMPT_TYPE_SUMMARIZE,
+			icon: postExcerpt,
+		},
+		{
+			name: __( 'Expand', 'jetpack' ),
+			key: QUICK_EDIT_KEY_MAKE_LONGER,
+			aiSuggestion: PROMPT_TYPE_MAKE_LONGER,
+			icon: postContent,
+		},
+	],
+	'core/list': [
+		{
+			name: 'Turn list into a table',
+			key: 'turn-into-table',
+			aiSuggestion: PROMPT_TYPE_USER_PROMPT,
+			icon: blockTable,
+			options: {
+				userPrompt: 'make a table from this list, do not enclose the response in a code block',
+			},
+		},
+		{
+			name: 'Write a post from this list',
+			key: 'write-post-from-list',
+			aiSuggestion: PROMPT_TYPE_USER_PROMPT,
+			icon: post,
+			options: {
+				userPrompt: 'Write a post based on the list items. Try to use a heading for each entry',
```

Comment:
the option key itself could have a less confusing name, we use userPrompt on other places of the code, so maybe we can use something more proper, like `presetPrompt`?